doc/classes/NavigationAgent.xml

```diff
@@ -0,0 +1,165 @@
+			<description>
+				Sets the user desired final location. This will clear the current navigation path.
+			</description>
+		</method>
+		<method name="set_velocity">
+			<return type="void">
+			</return>
+			<argument index="0" name="velocity" type="Vector3">
+			</argument>
+			<description>
+				Sends the passed in velocity to the collision avoidance algorithm. It will adjust the velocity to avoid collisions. Once the adjustment to the velocity is complete, it will emit the [signal velocity_computed] signal.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="agent_height_offset" type="float" setter="set_agent_height_offset" getter="get_agent_height_offset" default="0.0">
+			The agent height offset to match the navigation mesh height.
+		</member>
+		<member name="ignore_y" type="bool" setter="set_ignore_y" getter="get_ignore_y" default="true">
+			Ignores collisions on the Y axis. Must be true to move on a horizontal plane.
+		</member>
+		<member name="max_neighbors" type="int" setter="set_max_neighbors" getter="get_max_neighbors" default="10">
+			The maximum number of neighbors for the agent to consider.
+		</member>
+		<member name="max_speed" type="float" setter="set_max_speed" getter="get_max_speed" default="10.0">
+			The maximum speed that an agent can move.
+		</member>
+		<member name="neighbor_dist" type="float" setter="set_neighbor_dist" getter="get_neighbor_dist" default="50.0">
+			The distance to search for other agents.
+		</member>
+		<member name="path_max_distance" type="float" setter="set_path_max_distance" getter="get_path_max_distance" default="3.0">
+			The maximum distance the agent is allowed away from the ideal path to the final location. This can happen due to trying to avoid collisions. When the maximum distance is exceded, it recalculates the ideal path.
+		</member>
+		<member name="radius" type="float" setter="set_radius" getter="get_radius" default="1.0">
+			The radius of the agent.
+		</member>
+		<member name="target_desired_distance" type="float" setter="set_target_desired_distance" getter="get_target_desired_distance" default="1.0">
+			The distance threshold before a target is considered to be reached. This will allow an agent to not have to hit a point on the path exactly, but in the area.
+		</member>
+		<member name="time_horizon" type="float" setter="set_time_horizon" getter="get_time_horizon" default="5.0">
+			The minimal amount of time for which this agent's velocities, that are computed with the collision avoidance algorithim, are safe with respect to other agents. The larger the number, the sooner the agent will respond to other agents, but less freedom in choosing its velocities. Must be positive.
+		</member>
+	</members>
+	<signals>
+		<signal name="navigation_finished">
+			<description>
+				Notifies when the final location is reached.
+			</description>
+		</signal>
+		<signal name="path_changed">
+			<description>
+				Notifies when the navigation path changes.
```

Comment:
Might be useful to specify which kind of changes - is it changes done by the navigation system due to e.g. collision avoidance and/or the result of user calls to update configuration?